docs/source/user_guide/installation/install-source-windows.rst

```diff
@@ -0,0 +1,382 @@
+Install from source (Windows)
+=========================================
+
+.. contents::
+
+This page describes the build and installation of PyBaMM from the source code, available on GitHub. Note that this is **not the recommended approach for most users** and should be reserved to people wanting to participate in the development of PyBaMM, or people who really need to use bleeding-edge feature(s) not yet available in the latest released version. If you do not fall in the two previous categories, you would be better off installing PyBaMM using pip or conda.
+
+Lastly, familiarity with the Python ecosystem is recommended (pip, virtualenvs).
+Here is a gentle introduction/refresher: `Python Virtual Environments: A Primer <https://realpython.com/python-virtual-environments-a-primer/>`_.
+
+
+Prerequisites
+---------------
+
+To use PyBaMM, you must have Python 3.9, 3.10, 3.11, or 3.12 installed.
+
+To install Python 3.X, download the installation files from `Python’s
+website <https://www.python.org/downloads/windows/>`_. Make sure to
+tick the box on ``Add Python 3.X to PATH``. For more detailed
+instructions please see the `official Python on Windows
+guide <https://docs.python.org/3.9/using/windows.html>`__.
+
+(Optional) If you wish to build the documentation locally, you can install ``graphviz`` using the `Chocolatey <https://chocolatey.org/>`_ package manager,
+
+.. code:: bash
+
+	choco install graphviz --version=8.0.5
```

Comment:
Is this exact version required?

Reply:
I am not sure myself. I copied this from the `run_periodic_tests.yml`. I thought it might have a linked issue.